pkg/watermark/processor/entity.go

```diff
@@ -21,6 +21,12 @@ func (w Watermark) String() string {
 	return t.Format(time.RFC3339)
 }
 
+func (w Watermark) Unix() int64 {
+	var location, _ = time.LoadLocation("UTC")
```

Comment:
Do we need the location here? `time.Unix()` does not depend on the location associated with it.

Reply:
Thanks, I will remove.
> The Time.Unix() function in Go language is used to yield “t” as a Unix time that is the number of seconds passed from January 1, 1970, in UTC and the output here doesn’t rely upon the location connected with t.